src/components/tooltip/Tooltip.tsx

```diff
@@ -332,17 +332,17 @@ class Tooltip extends React.Component<TooltipProps, State> {
                 onContextMenu={this.handleTooltipEvent}
                 onKeyPress={this.handleTooltipEvent}
             >
-                <div role="tooltip" aria-live="polite" aria-hidden={isLabelMatchingTooltipText}>
+                <div aria-live="polite" aria-hidden={isLabelMatchingTooltipText}>
                     {tooltipInner}
                 </div>
             </div>
         ) : (
             <div
                 className={classes}
                 id={this.tooltipID}
-                role="tooltip"
                 aria-live="polite"
                 aria-hidden={isLabelMatchingTooltipText}
+                {...(theme === TooltipTheme.DEFAULT ? { role: 'tooltip' } : {})}
```

Comment:
What about `TooltipTheme.CALLOUT`? Should we just hide this role if the theme is `TooltipTheme.ERROR`?

Reply:
@laughnan You're right, I updated this PR.